umh-core/pkg/config/config.go

```diff
@@ -18,28 +18,31 @@ import (
 	"reflect"
 
 	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/config/benthosserviceconfig"
+	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/config/redpandaserviceconfig"
 	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/config/s6serviceconfig"
 
 	"github.com/tiendc/go-deepcopy"
 )
 
 type FullConfig struct {
-	Agent    AgentConfig     `yaml:"agent"`    // Agent config, requires restart to take effect
-	Services []S6FSMConfig   `yaml:"services"` // Services to manage, can be updated while running
-	Benthos  []BenthosConfig `yaml:"benthos"`  // Benthos services to manage, can be updated while running
-	Nmap     []NmapConfig    `yaml:"nmap"`     // Nmap services to manage, can be updated while running
+	Agent              AgentConfig               `yaml:"agent"`                        // Agent config, requires restart to take effect
+	Services           []S6FSMConfig             `yaml:"services,omitempty"`           // Services to manage, can be updated while running
+	Benthos            []BenthosConfig           `yaml:"benthos,omitempty"`            // Benthos services to manage, can be updated while running
+	Nmap               []NmapConfig              `yaml:"nmap,omitempty"`               // Nmap services to manage, can be updated while running
+	Redpanda           RedpandaConfig            `yaml:"redpanda,omitempty"`           // Redpanda config, can be updated while running
```

Comment:
the config / itnernal change needs to be done (probably causing merge conflicts), but should be quick